sourcecode/www/src/components/ResourcePage/components/ViewContainer.jsx

```diff
@@ -64,7 +64,7 @@ const ViewContainer = ({ children, showDeleteButton, resources }) => {
                 resource={selectedResource}
             />
             <Dialog
-                open={showConfirmDeletionModal}
+                open={!!showConfirmDeletionModal}
```

Comment:
Please use `Boolean(showConfirmDeletionModal)` here, it's easier on the brain.

Reply:
Fixed